dashboard/db/migrate/20190718184223_convert_level_source_id_to_bigint.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertLevelSourceIdToBigint < ActiveRecord::Migration[5.0]
+  def up
+    change_column :level_sources_multi_types, :level_source_id, :bigint
```

Comment:
Do we need some logic here to specify `BIGINT(11) UNSIGNED` like [previous migrations](https://github.com/code-dot-org/code-dot-org/blob/a87e399f0b9a3d896d44a669e781ae31d052296b/dashboard/db/migrate/20190517000300_change_integer_to_bigint_in_paired_user_levels.rb#L3)?
```suggestion
    change_column :level_sources_multi_types, :level_source_id, 'BIGINT(11) UNSIGNED'
```

Reply:
That was just for parity with ghost, right? I guess I can do that if that's what the ID field in `level_sources` will be ultimately...does it matter?

Reply:
We need parity with the datatype of the `user_levels.id` column because we don’t want `user_levels` to autoincrement to a value that is permitted with unsigned bigint, but is too large for the signed int column in this table.

Reply:
Understood on that point...looking at docs, I didn't realize that `unsigned` actually doubles the maximum integer value allowed for the column 🤦‍♂ -- will update.

Reply:
Updated! Rolled back and ran new migration.

Reply:
I know I just suggested using `BIGINT(11) UNSIGNED`, but are we sure this configures the datatype on the column exactly the same way that gh-ost set the datatype on the column in the `user_levels.id` column?  Doesn't gh-ost set it to bigint(20)?

Reply:
That does sound familiar, but I _think_ you can specify whatever you want in gh-ost (I just looked at the modify option in the scripts on `production-daemon` that conducted the migration on `paired_user_levels` and it looks like this: `MODIFY id bigint(11) unsigned not null auto_increment`. I also tried `describe user_levels` on the production DB and got type = `bigint(11) unsigned`